app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -226,7 +230,9 @@ def with_version4_vm_service(vm)
   end
 
   def highest_supported_api_version
-    supported_api_versions.sort.last
+    versions = supported_api_versions
+    return '3' if versions.empty?
```

Comment:
Maybe versions.sort.last || '3' would be shortwr

Reply:
I mean supported_api_versions.sort.last || '3'

Reply:
Yes, that makes sense, thanks @borod108, I am changing it.